### PR TITLE
Removing api discovery link (#275)

### PR DESCRIPTION
**Pull request by maxim-tschumak**
Related to #275.

### CODE REVIEW

File: api-operation/ApiOperation.md [hunk 1]
```diff
@@ -21,12 +21,10 @@ another application owned by a different team or even an external business partn
...
 for operational or implementation internal purposes.
 
-Background: In our dynamic and complex service infrastructure, it is important to provide API client
-developers a central place with online access to the OpenAPI reference definitions of all running applications.
-[API Discovery](https://github.com/zalando-incubator/api-discovery) is an API definition
-crawler of the Zalando platform infrastructure; it checks all running applications via the endpoint above and
-stores the discovered API definitions. API Discovery itself provides a RESTful API as well as an
-API Viewer (Swagger-UI) for central access to all discovered API definitions.
+Background: In our dynamic and complex service infrastructure, it is important to provide API client developers a central place with online access to the OpenAPI reference definitions of all running applications.
+As a part of Zalando infrastructire, API Discovery solution is used as an API definition crawler.
```
Comment (tkrop) [inline api-operation/ApiOperation.md]:
```
As a part of the Zalando infrastructure, the `.well-known/schema-discovery` endpoint is used by the API Discovery to detect all API definitions.
```

Comment (maxim-tschumak) [inline api-operation/ApiOperation.md]: done

**Comment by dehora** [comment]
-1. This doesn't fix the problem on what to put at the well-known link.

**Comment by duergner** [comment]
Would it make sense to simply add the expected structure of that `schema-discovery` document directly to the guidelines?

**Comment by tkrop** [comment]
@dehora Oh, I see. You are right. Here is the current information http://stups.readthedocs.io/en/latest/components/twintip.html, but what I do not know: will it be deleted in short too?

**Comment by dehora** [comment]
>  but what I do not know: will it be deleted in short too?

I don't know, but I can tell you elements of Stups are being moved away from. Unless Zalando's not running twintip internally it seems to make sense to inline the JSON structure or link to that page directly so people know what to do.

**Comment by dehora** [comment]
@duergner that option was mentioned here: https://github.com/zalando/restful-api-guidelines/issues/275#issuecomment-317723639. I don't really mind how it gets fixed.

**Comment by duergner** [comment]
If we can inline I would suggest to inline. Similar to what we did with JSON Fields.

**Comment by dehora** [comment]
> If we can inline I would suggest to inline.

I'm fine with this: @maxim-tschumak @tkrop,  thoughts?

**Comment by maxim-tschumak** [comment]
I agree with @duergner and @dehora

File: api-operation/ApiOperation.md [hunk 2]
```diff
@@ -5,10 +5,17 @@
 All service applications must support access to the OpenAPI Reference Definitions of their external APIs — it
 is optional for internal APIs — via the following two API endpoints:  
 
-* endpoint(s) for GET access on its OpenAPI definition(s), for instance
-  `https://example.com/swagger.json` or `https://example.com/swagger.yaml`.
-* API Discovery endpoint `https://example.com/.well-known/schema-discovery` that delivers
-  the OpenAPI definition endpoint(s) above (see the link below for a description of its format).
+* endpoint(s) for GET access on its OpenAPI definition(s), for instance `https://example.com/swagger.json` or `https://example.com/swagger.yaml`.
+* API Discovery endpoint `https://example.com/.well-known/schema-discovery` that delivers the following JSON discovery document:
```
Comment (ePaul) [inline api-operation/ApiOperation.md]: Maybe better "... that delivers a JSON discovery document of the following form (adapt the `schema_url` to point to the endpoint mentioned above):"

**Comment by dehora** [comment]
:+1:

**Comment by duergner** [comment]
👍

**Pull request merged**